### PR TITLE
Tweaked the docs and script to work better with Linux

### PR DESCRIPTION
**Pull request by bigbitbus**
This is to address the Ubuntu Linux install limitations of the curretn install script.

### CODE REVIEW

File: content/en/Installation/_index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Opta currently has the following system prerequisites to operate normally:
 - [GCP SDK](https://cloud.google.com/sdk/docs/install) (For GCP only)
 - [Azure Cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) (For Azure only)
 - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) (v2) (For AWS only)
+- Opta assumes that git, curl and unzip tools are already installed on the command line; in addition if you are using Ubuntu/Debian Linux then the `language-pack-en` package should be installed; you may also need to add `export LC_CTYPE=en_US.UTF-8` to your profile.
```
Comment (juandiegopalomino) [inline content/en/Installation/_index.md:21]: can we add them by doing
```bash
sudo apt-get update
sudo apt-get install git curl unzip
```
in the script

Comment (bigbitbus) [reply content/en/Installation/_index.md:21]: If we assume its only ubuntu/debian we want to address...the script right now is Linux distro agnostic.

**Review by NitinAgg — APPROVED** [review]

**Comment by bigbitbus** [comment]
Before I merge, @NitinAgg or @juandiegopalomino can you run the modified script on mac and confirm, I have confirmed it on Ubuntu as working.

**Comment by NitinAgg** [comment]
Will run the script on mac.

But we should solve this with CI eventually. Should we move this script to
opta repo?

File: static/install.sh [hunk 2]
```diff
@@ -72,3 +73,4 @@ else
 fi
 
 echo "Successfully installed! Now you can run it via invoking $RUNPATH"
+echo "To add opta to your path, please add" 'export PATH=$PATH:'"$RUNPATH" 'to you cli profile'
```
Comment (NitinAgg) [inline static/install.sh:76]: I see this even on mac. We should only show this if RUNPATH != opta?

**Review by juandiegopalomino — APPROVED** [review]

File: static/install.sh (hunk 2 above)
Comment (juandiegopalomino) [reply static/install.sh:76]: easy check to add, right?

Comment (bigbitbus) [reply static/install.sh]: pushed a change to limit this  path setting suggestion to only Linux

**Review by NitinAgg — APPROVED** [review]

**Comment by bigbitbus** [comment]
Just putting a note here for Centos/Redhat users of Opta - tested on CentOS Linux release 8.4.2105; you will need to install the prerequisites as mentioned in https://docs.opta.dev/installation/; in addition, for getting the terminal locale right issue these commands:

```
yum install -y glibc-locale-source
localedef --no-archive -i en_US -f UTF-8 en_US.UTF-8
export LANG=en_US.UTF-8
```

**Pull request merged**